### PR TITLE
Fund Tests Accounts in CI Pipeline

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Occasionally funds for e2e tests will run low. This PR adds a script to check balances prior to running e2e tests in the pipeline outputting their current balance and adding funds from the alfajores test faucet if funds are below 100 for CELO, cUSD, & cEUR. If other assets are added in the future they can be added to the Valora Test Faucet, `0xe5F5363e31351C38ac82DBAdeaD91Fd5a7B08846`, and distributed to the e2e test wallet with relatively minor changes.

### Other changes

Generate a `.env` file in `packages/mobile/test` when run in CI.

Adjusted e2e tests in CI pipeline to run the `./test/fund-e2e-accounts.ts` script using values set in the `.env` file

### Tested

Script was tested locally. Script was tested in CI by checking the output of e2e tests on this PR.

### How others should test

Check to see that balances are reported when e2e tests run on this PR

### CODE REVIEW

File: packages/mobile/e2e/src/usecases/OffRamps.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ export default offRamps = () => {
     })
 
     describe('When Address Selected', () => {
-      const randomAmount = `${Math.random().toFixed(3)}`
+      const randomAmount = `${(Math.random() * 10 ** -1).toFixed(3)}`
```
[the remark this answers is not in the document]
Comment (MuckT) [reply packages/mobile/e2e/src/usecases/OffRamps.js:121]: We can send back to the faucet, for now I've just set it up to fund 25 of a coin if the balance is lower than 100.

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -40,6 +40,16 @@ jobs:
...
+      - name: Create E2E Test .env File
+        env:
+          TEST_FAUCET_SECRET: ${{ secrets.TEST_FAUCET_SECRET }}
+        run : |
+          cd packages/mobile/test
+          echo TEST_FAUCET_SECRET=$TEST_FAUCET_SECRET >> .env
+      - name: Fund E2E accounts
+        run : |
+          cd packages/mobile/test
+          node ./fund-e2e-accounts.ts
```
[the remark this answers is not in the document]
Comment (MuckT) [reply .github/workflows/e2e.yml:52]: I was hoping to use the .env file for some of the verification flow tests secrets also.

[the remark this answers is not in the document]
Comment (MuckT) [reply .github/workflows/e2e.yml:52]: @jeanregisser would it be better to create the .env file in `packages/mobile/e2e/.env`?

[the remark this answers is not in the document]
Comment (jeanregisser) [reply .github/workflows/e2e.yml]: I see, should those secrets be accessible by Valora devs locally?
If so we, yes I think we should put them in `packages/mobile/e2e/.env.enc` and decrypt with the rest of files we have encrypted.

**Review by jeanregisser — COMMENTED** [review]
Small thing, would be good to move `packages/mobile/test/fund-e2e-accounts.ts` to `packages/mobile/scripts/fund-e2e-accounts.ts` (or `packages/mobile/e2e/scripts/fund-e2e-accounts.ts`?)
`packages/mobile/test` is only for unit tests things sofar.

File: .github/workflows/e2e.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (MuckT) [reply .github/workflows/e2e.yml]: Added to the decrypt script and committed the `packages/mobile/e2e/.env.enc`.

File: packages/mobile/.gitignore [hunk 3]
```diff
@@ -98,3 +98,6 @@ src/icons/DefaultAvatar.tsx
 src/images/*
 !src/images/Images.ts
 !src/images/link-arrow.png
+
+# E2E Test Secrets
+test/.env
```
Comment (jeanregisser) [inline packages/mobile/.gitignore:103]: Looks like this one is not necessary, since it was added to `packages/mobile/e2e/.gitignore`

**Review by jeanregisser — APPROVED** [review]
Great! 👍 
Already approved but see my comment.

File: packages/mobile/.gitignore (hunk 3 above)
Comment (MuckT) [reply packages/mobile/.gitignore]: Removed the unused line from the .gitignore in `packages/mobile/test/.gitignore`

**Pull request merged**